NuGet.config

```diff
@@ -2,7 +2,8 @@
 <configuration>
   <packageSources>
     <clear />
-    <add key="AspNetCore" value="https://dotnet.myget.org/F/aspnetcore-ci-dev/api/v3/index.json" />
+    <add key="AspNetCore" value="https://dotnet.myget.org/F/aspnetcore-master/api/v3/index.json" />
+    <add key="AspNetCoreRelease" value="https://dotnet.myget.org/F/aspnetcore-ci-release/api/v3/index.json" />    
```

Comment:
We will:
- hardcode "2.0.0" for Asp.NET Core packages and they will come from nuget.org (already done)
- hardcode "2.0.0" for Sources packages and get them from the aspnet-master feed 
- bring back the aspnet-ci-dev feed as it contains CoreFxLabs packages - (since the version for Asp.NET Core dependencies will have exact version we will not pick 2.1.0 dependencies) - we can continue to use `-*` for corefxlabs packages
- remove aspnet-ci-release feed